addon/bootstrap.js

```diff
@@ -51,8 +52,11 @@ const prefObserver = {
   }
 };
 
+let localePref = (prefs.getPrefType(LOCALE_PREF) && prefs.getCharPref(LOCALE_PREF)) || "en-US";
+
 function startup(data, reason) { // eslint-disable-line no-unused-vars
   console.log("bootstrap startup called");
+  Cu.import("chrome://screenshots/content/localizedTooltips.jsm");
```

Comment:
I read about it while in the midst of multiple import problems, so which things I needed to do and which I did because I was flailing around, I'm not sure.